webknossos/__generate_client.py

```diff
@@ -336,8 +420,9 @@ def set_response_schema_by_example(
         for path_method in path.values()
         if path_method["operationId"] == operation_id
     ][0]
-    make_properties_required(recorded_response_schema)
+    did_require = make_properties_required(recorded_response_schema)
```

Comment:
```suggestion
    did_require = make_properties_required_inplace(recorded_response_schema)
```
(just a suggestion, feel free to ignore. Coming from scala-land everything casually mutable scares me :D)

Reply:
The I'd have to set it for most functions, I'll leave it as-is for now, I double-checked that it's mentioned in all doc-strings.